lib/yggdrasil/plug/SkipIfAuthenticated.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.Plug.SkipIfAuthenticated do
```

Comment:
Perhaps a better name would be `RedirectIfAuthenticated`? It's unclear what `skip` would do.